packages/babel-plugin-syntax-import-assertions/src/index.ts

```diff
@@ -7,7 +7,7 @@ export default declare(api => {
     name: "syntax-import-assertions",
 
     manipulateOptions(opts, parserOpts) {
-      parserOpts.plugins.push(["importAssertions"]);
+      parserOpts.plugins.push("importAssertions");
```

Comment:
They are equivalent, but the type-checker favors the latter form, which is also more common.